dash_bio/Molecule3dViewer.py

```diff
@@ -17,18 +17,21 @@ class Molecule3dViewer(Component):
 - modelData (optional): The data that will be used to display the molecule in 3D
 The data will be in JSON format
 and should have two main dictionaries - atoms, bonds. modelData has the following type: dict containing keys 'atoms', 'bonds'.
-Those keys have the following types:
+Those keys have the following types: 
```

Comment:
I'm sure you'll fix it later, but I think it shouldn't have landed in a commit in the first place...

Reply:
@mkcor - This comes from running `npm run build:all`, I plan to do a rebase of this PR, and maybe even break this PR in smaller ones and delete this one.

Reply:
Ok. Looks like I'll have to study what's happening in this `build` at some point... #214